Kasulejoseph/fast-food-fast-db
# #160819084 Ft order history to develop

**What this PR do?**
 -  Add order history and menu end points 
**Description of tasks to be completed***
 - A user should be able to add to view all the his previous orders
given he browser the end point `/api/v1/orders`.
- The PR also add a functionality of the admin to add `GET /api/v1/menu` a menu and 
users to view all the posted menu `POST /api/v1/menu`
-**How to manually test**
- Log in as a user in postman
- Browse the route `localhost:5000/api/v1/orders` and your previous orders will be
displayed in case you have
*To add menu*
- Login as an admin, insert menu fields and post
*To see menu*
- you create a get request on `/api/v1/menu`
**Relevant Stories**
#160822749
#160819084

**Screenshots**
1. order history
![orderhist](https://user-images.githubusercontent.com/32167860/46409779-665bb580-c71f-11e8-8db8-617058059866.png)

2. menu
![onmenu](https://user-images.githubusercontent.com/32167860/46409875-b470b900-c71f-11e8-8952-c8261d8d0b5e.png)
